src/InverseSolve/models.jl

```diff
@@ -25,6 +25,11 @@ function fMod(m;f::Function=identity,df::Function=m->ones(length(m)))
 	return f(m),sdiag(df(m))
 end
 
+function dummyMod(m)
```

Comment:
My julia version doesn't seem to know this.
julia> DiagonalScaling(1.0)
ERROR: UndefVarError: DiagonalScaling not defined

julia> diagonalScaling(1.0)
ERROR: UndefVarError: diagonalScaling not defined

On Mon, Apr 18, 2016 at 2:02 PM, Lars Ruthotto <notifications@github.com>
wrote:

> In src/InverseSolve/models.jl
> <https://github.com/JuliaInv/jInv.jl/pull/3#discussion_r60133700>:
>
> > @@ -25,6 +25,11 @@ function fMod(m;f::Function=identity,df::Function=m->ones(length(m)))
> >  	return f(m),sdiag(df(m))
> >  end
> >
> > +function dummyMod(m)
>
> thanks for the explanation. I think what we should use is
>
> DiagonalScaling(1.0)
>
> This is treated like a sparse identity (of any size) also, for example,
> when adding it to another square sparse matrix. How about changing the name
> of your method from dummyMod to something more meaningful, for example,
> idMod?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaInv/jInv.jl/pull/3/files/ace570ed0d40a4b42df3ab8ff7ed285f6dba44f0#r60133700>
>